libs/libsigar/Makefile

```diff
@@ -0,0 +1,43 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=libsigar
+PKG_VERSION:=1.6.4.1
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/boundary/sigar.git
+PKG_SOURCE_VERSION:=a6c61edf8c64e013411e8c9d753165cd03102c6e
+PKG_MIRROR_HASH:=3a2bd947e55a93ced7f56953ee6cf15a7bc7679d1e0f9b97a10f05cd3909c533
+
+PKG_LICENSE:=Apache-2.0
+PKG_LICENSE_FILE:=LICENSE
+
+PKG_BUILD_DEPENDS:=libtirpc
+
+include $(INCLUDE_DIR)/package.mk
+include $(INCLUDE_DIR)/cmake.mk
+
+PKG_BUILD_PARALLEL:=1
+CMAKE_OPTIONS:=-DHZ=100
+EXTRA_CFLAGS:=-fgnu89-inline -DHZ=100 -I$(STAGING_DIR)/usr/include/tirpc -ltirpc
```

Comment:
I would avoid -fgnu89-inline. In my experience, -std=gnu89 is better as it generates fewer warnings with the packages that I've seen.

The alternative is to fix the actual errors by adding static or what have you.

Reply:
Turns out after switching to the boundary fork, this option is no longer needed.